db/init/oracle/schema.sql

```diff
@@ -2997,4 +2997,27 @@ comment on column SCALE_HISTORY.msg
 comment on column SCALE_HISTORY.date_created
     is 'create time';
 comment on column SCALE_HISTORY.date_updated
-    is 'update time';
\ No newline at end of file
+    is 'update time';
+
+
+create table namespace_user_rel
+(
+    id             varchar(128) not null,
+    namespace_id   varchar(50) NOT NULL,
+    user_id        varchar(128) NOT NULL,
+    date_created   timestamp(3) default SYSDATE not null,
+    date_updated   timestamp(3) default SYSDATE not null,
+    PRIMARY KEY (id)
+);
```

Comment:
you shoud add the script to upgrade script and you should add this to h2 sql

Reply:
oh, i missed